syncd/syncd.cpp

```diff
@@ -65,6 +65,13 @@ std::set<sai_object_id_t> initViewRemovedVidSet;
  */
 volatile bool g_asicInitViewMode = false;
 
+#ifdef SAITHRIFT
+/*
+ * SAI switch global needed for RPC server
+ */
+sai_object_id_t gSwitchId;
```

Comment:
What component does use the gSwitchId variable ?
I see you assign some value to it below, but how it is used?

Reply:
It is required by the latest SAI thrift implementation:
https://github.com/opencomputeproject/SAI/blob/master/test/saithrift/src/switch_sai_rpc_server.cpp#L80

Reply:
I'd better to make interface of SAI thrift implementation more explicit by introducing 
switch_id parameter to start_sai_thrift_rpc_server() function.
But anyway.